tests/test_ag3.py

```diff
@@ -730,10 +730,10 @@ def test_snp_allele_frequencies__str_cohorts():
     cohorts = "admin1_month"
     min_cohort_size = 10
     universal_fields = [
-        "contig",
-        "position",
-        "ref_allele",
-        "alt_allele",
+        # "contig",
+        # "position",
+        # "ref_allele",
+        # "alt_allele",
```

Comment:
I'd delete these rather than leave commented out. 

Also I'd suggest adding a test on the index names.